docs/sdks/server-sdks/node.md

```diff
@@ -31,40 +31,11 @@ npm install @eppo/node-server-sdk
 </TabItem>
 </Tabs>
 
-## 2. Initialize the SDK
-
-Initialize the SDK with a SDK key, which can be generated in the Eppo interface. Initialization should happen when your application starts up to generate a singleton client instance, once per application lifecycle:
-
-```javascript
-import { init } from "@eppo/node-server-sdk";
-
-await init({
-  apiKey: "<API_KEY>",
-  assignmentLogger,
-});
-```
-
-After initialization, the SDK begins polling Eppo’s API at regular intervals to retrieve the most recent experiment configurations such as variation values and traffic allocation. The SDK stores these configurations in memory so that assignments thereafter are effectively instant. For more information, see the [architecture overview](/sdks/overview) page.
-
-If you are using the SDK for experiment assignments, make sure to pass in an assignment logging callback (see [section](#define-an-assignment-logger-experiment-assignment-only) below).
-
-### Initialization options
-
-How the SDK fetches experiment configurations is configurable via additional optional initialization options:
-
-| Option | Description | Default |
-| ------ | ----------- | ------- | 
-| **`requestTimeoutMs`** (number) | Timeout in milliseconds for HTTPS requests for the experiment configurations. | `5000` |
-| **`numInitialRequestRetries`** (number) | Number of _additional_ times the initial configurations request will be attempted if it fails. This is the request typically synchronously waited (via `await`) for completion. A small wait will be done between requests. | `1` |
-| **`pollAfterFailedInitialization`** (boolean) | Poll for new configurations even if the initial configurations request failed. | `false` |
-| **`throwOnFailedInitialization`** (boolean) | Throw an error (reject the promise) if unable to fetch initial configurations during initialization. | `true` |
-| **`numPollRequestRetries`** (number) | If polling for updated configurations after initialization, the number of additional times a request will be attempted before giving up. Subsequent attempts are done using an exponential backoff. | `7` |
-
-### Define an assignment logger (experiment assignment only)
+## 2. Define an assignment logger
 
 If you are using the Eppo SDK for experiment assignment (i.e randomization), pass in a callback logging function to the `init` function on SDK initialization. The SDK invokes the callback to capture assignment data whenever a variation is assigned.
```

Comment:
I've always found this sentence confusing. You are probably initializing the SDK once at app startup. Wouldn't you generally want to use the SDK for both feature gates and experiments? 

I think we should always have them pass in a logger. We've discussed adding more flexibility into when we do or don't invoke the logger that we can add later once fully supported

Let's instead use this section to show that BYO-logger is a good thing, not a feature gap. Something like:

Eppo encourages centralizing application logging as much as possible. Accordingly, instead of implementing a new logging framework, Eppo's SDK integrates with your existing logging system via a logging callback function defined at SDK initialization.

Reply:
updated